src/MetaCompilation/MetaCompilation/MetaCompilation/CodeFixProvider.cs

```diff
@@ -687,7 +687,8 @@ public sealed override FixAllProvider GetFixAllProvider()
             SyntaxGenerator generator = SyntaxGenerator.GetGenerator(document);
 
             var ifBlockStatements = new SyntaxList<StatementSyntax>();
-            StatementSyntax ifStatement = CodeFixHelper.TriviaCheckHelper(generator, declaration.Body, ifBlockStatements) as StatementSyntax;
+            var leadingTrivia = SyntaxFactory.TriviaList(SyntaxFactory.EndOfLine("\r\n"), SyntaxFactory.ParseLeadingTrivia("// Checks if there is any trailing trivia associated with the if-keyword").ElementAt(0), SyntaxFactory.EndOfLine("\r\n"));
```

Comment:
@zoepetard Instead of EndOfLine("\r\n"), use SyntaxFactory.CarriageReturnLineFeed